--- /home/ovirt/ovirt-engine/share/ovirt-engine/setup/plugins/ovirt-engine-common/base/core/misc.py	2016-08-31 13:42:21.000000000 +0000
+++ /home/ovirt/ovirt-engine/share/ovirt-engine/setup/plugins/ovirt-engine-common/base/core/misc.py	2016-08-31 20:25:58.947439383 +0000
@@ -82,7 +82,7 @@
 
         self.environment.setdefault(
             osetupcons.CoreEnv.DEVELOPER_MODE,
-            None
+            True
         )
         self.environment.setdefault(
             osetupcons.CoreEnv.UPGRADE_SUPPORTED_VERSIONS,
--- /home/ovirt/ovirt-engine/share/ovirt-engine/services/ovirt-engine/ovirt-engine.xml.in	2016-08-31 13:42:16.000000000 +0000
+++ /home/ovirt/ovirt-engine/share/ovirt-engine/services/ovirt-engine/ovirt-engine.xml.in	2016-08-31 20:31:12.106784608 +0000
@@ -162,18 +162,14 @@
       <logger category="org.ovirt" use-parent-handlers="false">
         <level name="INFO"/>
         <handlers>
-          <handler name="ENGINE"/>
-          <handler name="ovirt-logger"/>
-          {% if config.getboolean('ENGINE_LOG_TO_CONSOLE') %}
           <handler name="CONSOLE"/>
-          {% endif %}
         </handlers>
       </logger>
       <!-- Loggers for the UI -->
       <logger category="org.ovirt.engine.ui.frontend.server.gwt.OvirtRemoteLoggingService" use-parent-handlers="false">
         <level name="INFO"/>
         <handlers>
-          <handler name="UI"/>
+		  <handler name="CONSOLE"/>
         </handlers>
       </logger>
        <logger category="org.ovirt.engine.core.bll">
@@ -186,8 +182,7 @@
       <root-logger>
         <level name="INFO"/>
         <handlers>
-          <handler name="ovirt-logger"/>
-          <handler name="SERVER"/>
+          <handler name="CONSOLE"/>
         </handlers>
       </root-logger>
     </subsystem>
